rust/sdk-processor/src/utils/starting_version.rs

```diff
@@ -61,13 +61,19 @@ pub async fn get_min_last_success_version_parquet(
             .context("Failed to get minimum last success version from DB")?
     };
 
-    // If nothing checkpointed, return the `starting_version` from the config, or 0 if not set.
-    Ok(min_processed_version.unwrap_or(
-        indexer_processor_config
-            .transaction_stream_config
-            .starting_version
-            .unwrap_or(0),
-    ))
+    let config_starting_version = indexer_processor_config
+        .transaction_stream_config
+        .starting_version
+        .unwrap_or(0);
+
+    if let Some(min_processed_version) = min_processed_version {
+        Ok(std::cmp::max(
+            min_processed_version + 1,
```

Comment:
if `min_processed_version` = `config_starting_version` = 0, then this will return 1 so version 0 will never be processed. Removing the `+ 1` will fix this!

Reply:
good catch! will fix tjos

Reply:
https://github.com/aptos-labs/aptos-indexer-processors/blob/3dfd2f6798b7cf5b0b47d6ce3f92343c61a385cd/rust/sdk-processor/src/utils/starting_version.rs#L26-L43

hmm, in that case, would this also return 1?

Reply:
```
  // return the higher of the checkpointed version + 1 and `starting_version`.
    Ok(status.map(|status| {
        std::cmp::max(
            status.last_success_version as u64 + 1,
            indexer_processor_config
                .transaction_stream_config
                .starting_version
                .unwrap_or(0),
        )
    }))
    ```

Reply:
yeah lol do you mind changing that too please